lib/elixir/lib/kernel.ex

```diff
@@ -855,9 +855,25 @@ defmodule Kernel do
   end
 
   @doc """
-  Returns `true` if `term` is a map; otherwise returns `false`.
+  Returns `true` if `term` is a map, otherwise returns `false`.
 
   Allowed in guard tests. Inlined by the compiler.
+
+  > #### Structs are maps {: .info}
+  >
+  > Structs are also maps, and many of Elixir data structures are implemented
```

Comment:
```suggestion
  > Structs are also maps, and many of Elixir's data structures are implemented
```

Reply:
@vanderhoop I quickly grepped the codebase and seems we've been using `Elixir data structures` in other places so will keep the consistency (or we should fix it elsewhere too). Thank you!

Reply:
👍 🚀